esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -4108,7 +4108,10 @@ void CWsWuFileHelper::sendWUComponentLogStreaming(CHttpRequest* request, CHttpRe
                 (opt == CWUFileDownloadOption_GZIP) ? "tar" : "zip");
 
         //Send the zipped file and clean the workingFolder.
-        response->setContent(createIOStreamWithFileName(zipFileNameWithPath, IFOread));
+        auto stream = createIOStreamFromFile(zipFileNameWithPath, IFOread);
```

Comment:
I'm not a huge fan of auto when it's just used as a shortcut for a known type.  It's very helpful for template based and other obscure return values though.

Reply:
Sorry, I copied from https://github.com/hpcc-systems/HPCC-Platform/pull/15974. I will change it to IFileIOStream*.